File: String/Dcrpt-String/dcrpt_string.py
def decrypt_string(s, num):
    result = ""
    total = 0
    for i in range(1, len(s), 2):
        count = int(s[i])
        total += count
        if total >= num:
            result = s[i - 1]
            break
    print(result)

# Example usage
decrypt_string("a2b2c3d1", 8)

File: String/Dcrpt-String/DcrptString.js
function decryptString(str, num) {
  let sum = 0
  let result = ''

  for (let i = 1; i < str.length; i += 2) {
    let count = parseInt(str[i])
    sum += count
    if (sum >= num) {
      result = str[i - 1]
      break
    }
  }

  console.log(result)
}

// Example usage
decryptString('a2b2c3d1', 8)
